src/libcore/marker.rs

```diff
@@ -60,13 +76,16 @@ pub trait Sized {
 }
 
 /// Types that can be "unsized" to a dynamically sized type.
+///
+/// For example, the sized array type `[i8; 2]` implements `Unsize<[u8]>` and
```

Comment:
It might be worth briefly talking about how `Unsize` is paired with `CoerceUnsized` and what it's used for here.